### PR TITLE
Restructured the dir structure of project reality

### PR DESCRIPTION
**Pull request by saiakshith**
@veera2211  add this new project reality into our repository. It will be a good addition to our project.

### CODE REVIEW

File: src/main/java/com/reality/Life.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    private Health wellBeing;
+    private Girlfriend wifeMaterial;
+    private Children hyperActive;
+//    private Wife caringAndCrazy;
+    private Dog forHappiness;
+    private Home aPlaceOfLove;
+    private Friend notManyButWhomYouCanTrust;*/
+
+    public static void main(String[] args) {
+        System.out.println("Hello World !");
```
Comment (veera2211) [inline src/main/java/com/reality/Life.java]: Change the "Hello World" to "Chamma Chekka", Ok.

File: src/main/java/com/reality/Life.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+public class Life {
+/*    private Determination determination;
+    private Health wellBeing;
+    private Girlfriend wifeMaterial;
+    private Children hyperActive;
+//    private Wife caringAndCrazy;
+    private Dog forHappiness;
+    private Home aPlaceOfLove;
+    private Friend notManyButWhomYouCanTrust;*/
```
Comment (veera2211) [inline src/main/java/com/reality/Life.java]: Uncomment these lines.

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+<h2>Hello World!</h2>
```
Comment (veera2211) [inline src/main/webapp/index.jsp]: Replace "Hello World" to "Chamma", Ok.

**Review by veera2211 — COMMENTED** [review]
Please make those requested changes @saiakshith.

File: src/main/java/com/reality/Life.java (hunk 1 above)
Comment (saiakshith) [reply src/main/java/com/reality/Life.java]: "Chamma Chekka" is not good, so i added "My World!"

File: src/main/java/com/reality/Life.java (hunk 2 above)
Comment (saiakshith) [reply src/main/java/com/reality/Life.java]: I haven't created those classes yet, i will uncomment them once i have created those.

File: src/main/webapp/index.jsp (hunk 3 above)
Comment (saiakshith) [reply src/main/webapp/index.jsp]: "Chamme" is not good, so i made it "Your World!"

File: src/main/java/com/reality/Life.java (hunk 2 above)
Comment (veera2211) [reply src/main/java/com/reality/Life.java]: ok create those classes first @saiakshith

Thanks
@veera2211

**Review by veera2211 — CHANGES_REQUESTED** [review]
@saiakshith please make changes

**Pull request closed without merging**